### ISSUE TITLE
`Assert::notNull()` should warn when argument is never null

### ISSUE THREAD

**Issue by Creeperface01** [phpstan/phpstan-webmozart-assert#154]
The same issue applies probably to more checks than just this.

In this case $a is never going to be null but phpstan won't show an error
```php
$a = 1;

Assert::notNull($a);
```

**Comment by ondrejmirtes** [comment]
It does. Make sure to install phpstan-strict-rules, or enable this option: https://phpstan.org/config-reference#checkalwaystruechecktypefunctioncall

**Comment by herndlm** [comment]
and we're making sure that this works via https://github.com/phpstan/phpstan-webmozart-assert/blob/1.2.2/tests/Type/WebMozartAssert/ImpossibleCheckTypeMethodCallRuleTest.php#L20 😊
